usr/src/uts/i86pc/os/intr.c

```diff
@@ -471,6 +472,7 @@
 #include <sys/hypervisor.h>
 #endif
 
+boolean_t kernel_preempt_enable = B_FALSE;
```

Comment:
I guess my problem is that we're not really enabling kernel preemption in any sense, we're just making runrun act like kprunrun in a subset of cases.  Which is hard to come up with a great name for, certainly.  It's also hard to see why it's the right thing to be doing.